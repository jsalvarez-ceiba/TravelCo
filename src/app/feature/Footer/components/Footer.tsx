import React from 'react';

const Footer = () => {
  return (
    <div className="flexCenter bg-dark">
      <small className="text-white">
        TravelCo | Todos los derechos reservados 2021
      </small>
    </div>
  );
};

export default Footer;
