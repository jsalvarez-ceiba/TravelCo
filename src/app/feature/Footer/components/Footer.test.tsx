import React from 'react';
import {shallow} from 'enzyme';
import Footer from './Footer';
describe('Footer Component Test', () => {
    it('render', () => {

        const wrapper = shallow(<Footer/>);
        expect(wrapper).toMatchSnapshot();

    })
})