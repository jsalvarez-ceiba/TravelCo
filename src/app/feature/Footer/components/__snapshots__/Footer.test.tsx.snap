// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Test render 1`] = `
<div
  className="flexCenter bg-dark"
>
  <small
    className="text-white"
  >
    TravelCo | Todos los derechos reservados 2021
  </small>
</div>
`;
